tensorflow/python/training/monitored_session.py

```diff
@@ -575,7 +575,7 @@ def _close_internal(self, exception_type=None):
           ops.get_default_graph()._unsafe_unfinalize()  # pylint: disable=protected-access
 
   def _is_closed(self):
-    """Return True if the supervised session is closed.  For tests only.
+    """Return True if the monitored session is closed.  For tests only.
```

Comment:
That being said, it's not clear the documentation would be improved by keeping it the same, so i'll just merge this.